controllers/storage/minio.go

```diff
@@ -41,21 +51,30 @@ func (m *MinIOReconciler) Reconcile() (*lcm.CRStatus, error) {
 		return m.ProvisionExternalStorage()
 	}
 
-	err := m.KubeClient.Get(m.Ctx, m.getminIONamespacedName(), &minioCR)
+	err := m.KubeClient.Get(m.getMinIONamespacedName(), &minioCR)
 	if k8serror.IsNotFound(err) {
-		// TODO need test
 		return m.Provision()
 	} else if err != nil {
 		return minioNotReadyStatus(ErrorReason0, err.Error()), err
 	}
 
+	var minioStatefulSet appsv1.StatefulSet
```

Comment:
Why do you need to care about minioStatefulSet, but not minioCR?

Reply:
now the status in MinIO CR is not accurate, so we get status in minioStatefulSet to check the minIO is ready or not.

Reply:
OK, I think you can use a method to check whether the minIO is ready or not. In the `Reconcile`, it does not need to care about the concrete workload(statefulset).